github/pulls.go

```diff
@@ -251,12 +251,14 @@ type PullRequestMergeResult struct {
 
 // PullRequestOptions lets you define how a pull request will be merged.
 type PullRequestOptions struct {
```

Comment:
Could you please add `CommitTitle` to this struct?
https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button